WordPress/Classes/ViewRelated/Site Creation/Shared/TitleSubtitleTextfieldHeader.swift

```diff
@@ -60,7 +60,7 @@ private final class SearchTextField: UITextField {
 
         backgroundColor = .white
         clearButtonMode = .whileEditing
-        font = WPStyleGuide.fontForTextStyle(.headline)
+        font = WPStyleGuide.fontForTextStyle(.body, fontWeight: UIFont.Weight.regular)
```

Comment:
Would you consider removing `UIFont.Weight` here?

That is to say, this should be sufficient:
```
font = WPStyleGuide.fontForTextStyle(.body, fontWeight: UIFont.Weight.regular)
```

Reply:
Ah good point, thanks! Cleaned up.